### ISSUE TITLE
Deprecate the import wizard on the settings page

### ISSUE THREAD

**Issue by alachaum** [KeypDemoCorp/awesome-api-backend#344]
Accessibility should be considered when implementing this.

**Comment by pixdemo** [comment]
Has anyone confirmed whether this happens in production too?
